### DISCUSSION TITLE
Add the ability to rerun Jest tests from Allure TestOps

### DISCUSSION THREAD

**Discussion by yaroslavchaadaev** [allure-framework/allure2#2447] [Questions & Support] [answered]
Hello! Could you tell me, is it possible to add the ability to rerun Jest tests from Allure TestOps? Or increase the priority for the development of this task. Thanks. :)

**Comment by yaroslavchaadaev** [comment]
@epszaw 🙏🏻

**Reply by epszaw** [reply, marked as answer]
Hey there! Currently, the integration doesn't support test plans. We work on [new architecture research for `allure-js`](https://github.com/allure-framework/allure-js/pull/909) integrations and, I think, better implement test plan support after the research, using the new approach's benefits. But, if we stuck with something, or the research will take more, than we expected, I'll try to figure out, how we can make jest doing that.
